sema4ai/src/sema4ai_code/_lint.py

```diff
@@ -197,6 +197,26 @@ def _do_lint(self) -> None:
             self._lsp_messages.publish_diagnostics(doc_uri, found)
             return
 
+        if doc_uri.endswith("agent-spec.yaml"):
```

Comment:
There is `PackageYamlName` enum in `protocols.py` you could use, to avoid hardcoding the name ^^

Reply:
Done